QRCode.class.php

```diff
@@ -340,9 +340,9 @@ public function get($iSize = 150, $sECLevel = 'L', $iMargin = 1)
      *
```

Comment:
Can you please update the docBlock with `@param int $iSize`?

Reply:
Now better